src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1166,16 +1172,17 @@ public static class GuiceHolder implements LifecycleComponent {
         private static RepositoriesService repositoriesService;
         private static RemoteClusterService remoteClusterService;
         private static IndicesService indicesService;
-
         private static PitService pitService;
+        private static ExtensionsManager extensionsManager;
```

Comment:
I don't think this is used, lets remove

Reply:
I removed this for now, it will be needed for grabbing the extension information later though.